LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -188,7 +188,16 @@ public bool HasPassword(User user)
         /// <inheritdoc />
         public Task ChangePassword(User user, string newPassword)
         {
-            throw new NotImplementedException();
+            if (!LdapPlugin.Instance.Configuration.AllowPassChange)
+            {
+                throw new NotImplementedException();
+            }
+
+            var passAttr = LdapPlugin.Instance.Configuration.LdapPasswordAttribute;
+            var ldapUser = LocateLdapUser(user.Username) ?? throw new LdapException("No users found in LDAP Query");
+            using var ldapClient = ConnectToLdap() ?? throw new AuthenticationException("Failed to Connect or Bind to server");
```

Comment:
```suggestion
            using var ldapClient = ConnectToLdap();
```

ldapClient will never be null